### PR TITLE
feat: Snapshot page and Performance management page

### PR DESCRIPTION
**Pull request by Udit-takkar**
**Description**

This PR fixes #

- [x] Improve mobile version ( font size and image size)
- [x] gradient design
- [x] add animations

Screenshot 2023-06-18 at 3 10 20 PM

Floating animation

https://github.com/layer5io/layer5/assets/53316345/5c0f5cbd-df8e-4912-bf11-969a6be5c06e

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 758fb14bcad3f9dbc40d657815e11d6e6d410879 at: https://648ed4b573a33f1653c9dfa2--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 800ac36320d682503fd811bda5821a5519df2f47 at: https://6493645918764f17e772f802--layer5.netlify.app

**Comment by leecalcote** [comment]
@Shivam-AfA feedback?

**Comment by l5io** [comment]
🚀 Preview for commit 153e095bc559e2c84fc802e57d7579df2c50cfac at: https://64b541908d46fb02c17f24b7--layer5.netlify.app

**Review by Chadha93 — CHANGES_REQUESTED** [review]
Looks great @Udit-takkar 
Few changes are required.
1. Button padding is large.
2. Font weight.
3. Color should be black with yellow buttons.
4. Link Snapshot page to this section MeshMap GitHub Action (add button)

**Comment by Udit-takkar** [comment]
@Chadha93 
1) I didn't made any change in padding. I just used the Button component. It's pretty much the same like in design https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?node-id=15450%3A23535&mode=dev.

2)Fixed

3) It's white and yellow design

4) I didn't this point. where is this add button?

**Comment by l5io** [comment]
🚀 Preview for commit 715711426ceceb1d9736751cd14c9979b8ae627e at: https://64bd3d2bc252e02d3a126f5e--layer5.netlify.app

**Comment by Chadha93** [comment]
> @Chadha93
> 
> 1. I didn't made any change in padding. I just used the Button component. It's pretty much the same like in design [figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?node-id=15450%3A23535&mode=dev](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?node-id=15450%3A23535&mode=dev).
> 
> 2)Fixed
> 
> 3. It's white and yellow design
> 4. I didn't this point. where is this add button?

Thanks @Udit-takkar. 
1,3. @ritiksaxena124 the colors and padding needs to be consistent in design.
4. @Udit-takkar I meant, add a new button in the section to navigate to snapshot page. "MeshMap GitHub Action". Yes you are right, it's not in design let's fix that too.

**Comment by Chadha93** [comment]
@Udit-takkar merge conflicts....

**Comment by Udit-takkar** [comment]
@Chadha93 Okay will fix the conflicts

File: src/pages/cloud-native-management/gitops/performance-management.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const Page = () => {
+  return (
+    <>
+      <PerformanceManagementPage />
+    </>
+  );
+};
+export default Page;
+export const Head = () => {
+  // TODO Change this
```
Comment (Udit-takkar) [inline src/pages/cloud-native-management/gitops/performance-management.js:15]: @Chadha93 what to add in title and description?

File: src/pages/cloud-native-management/gitops/snapshot.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  return (
+    <>
+      <SnapshotPage />
+    </>
+  );
+};
+export default Meshery;
+export const Head = () => {
+  // TODO Change this
+  return  <SEO title="GitOps with Meshery"
```
Comment (Udit-takkar) [inline src/pages/cloud-native-management/gitops/snapshot.js:16]: @Chadha93 what to add in title and description?

**Comment by l5io** [comment]
🚀 Preview for commit 4cc5c95ef8e5944881e11894792cb714a71e2c18 at: https://64cc09ff74696f11384b9c7e--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 40215426b4f1617520a2c877382bf401a8136597 at: https://64ceb332759c686ca30303de--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 8a16abf5d987655bfcdea44ad45559edc226ac6e at: https://64cec0f2be3d0c6d4d0eb365--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit fdfb772df3d398e51fbba194f81f523b659ac73d at: https://64cf5d16842a6d306897c50b--layer5.netlify.app

**Comment by leecalcote** [comment]
I'm not sure what was taking us (including me) so long to get this merged, but it is now!

**Comment by Udit-takkar** [comment]
@leecalcote this is still pending https://github.com/layer5io/layer5/pull/4396/files#r1283661565

File: src/pages/cloud-native-management/gitops/performance-management.js (hunk 1 above)
Comment (leecalcote) [reply src/pages/cloud-native-management/gitops/performance-management.js:9]: Uh-oh. I never delivered in this to @Udit-takkar. Lee has an outstanding action item here.

**Pull request merged**